pkg/cgu/cgu.go

```diff
@@ -26,6 +33,94 @@ type CguBuilder struct {
 	errorMsg string
 }
 
+// NewCguBuilder creates a new instance of CguBuilder.
+func NewCguBuilder(
+	apiClient *clients.Settings,
+	name, nsname string,
+	maxConcurrency int) *CguBuilder {
+	glog.V(100).Infof(
+		"Initializing new CGU structure with the following params: %s, %s",
```

Comment:
Please add maxConcurrency to the log message

Reply:
Added.